### ISSUE TITLE
Stuck on Mac with remote files

### ISSUE THREAD

**Issue by wabiloo** [vivictorg/vivictpp#23]
Hi,

Just installed vivict++ on my Mac (M1 Pro). I was hoping to be able to use 2 remote streams (HLS via https), but found that I only got the first frame, and then none of the keyboard bindings work.

```
vivictpp https://.../video1.m3u8 https://..../video2.m3u8
```

Any reason why that wouldn't work?

**Comment by grusell** [comment]
I just tested, seems there is a problem with playback for hls live playlists. I got the same behavior that you described above. I will look into this as soon as possible.

**Comment by grusell** [comment]
Fix for playing hls live manifests is now in dev branch. If you either build from that branch or install with brew using the `--HEAD` option you should hopefully be able to watch your hls streams.

I will make a proper release in the next few days.

**Comment by wabiloo** [comment]
brew uninstalled previous version
brew installed --HEAD version

Then on running it:
```
dyld[76872]: Library not loaded: '@rpath/libsdl2.dylib'
  Referenced from: '/opt/homebrew/Cellar/vivictpp/HEAD-1b6c5e3/bin/vivictpp'
  Reason: tried: '/opt/homebrew/Cellar/vivictpp/HEAD-1b6c5e3/bin/subprojects/SDL2-2.24.0/libsdl2.dylib' (no such file), '/opt/homebrew/Cellar/vivictpp/HEAD-1b6c5e3/bin/subprojects/SDL2_ttf-2.0.12/libsdl2.dylib' (no such file), '/opt/homebrew/Cellar/vivictpp/HEAD-1b6c5e3/bin/subprojects/fmt-9.0.0/libsdl2.dylib' (no such file), '/opt/homebrew/opt/freetype/lib/libsdl2.dylib' (no such file), '/opt/homebrew/Cellar/ffmpeg/5.1.2/lib/libsdl2.dylib' (no such file), '/opt/homebrew/Cellar/libx11/1.8.1/lib/libsdl2.dylib' (no such file), '/opt/homebrew/Cellar/libxext/1.3.4/lib/libsdl2.dylib' (no such file), '/opt/homebrew/Cellar/vivictpp/HEAD-1b6c5e3/bin/subprojects/SDL2-2.24.0/libsdl2.dylib' (no such file), '/opt/homebrew/Cellar/vivictpp/HEAD-1b6c5e3/bin/subprojects/SDL2_ttf-2.0.12/libsdl2.dylib' (no such file), '/opt/homebrew/Cellar/vivictpp/HEAD-1b6c5e3/bin/subprojects/fmt-9.0.0/libsdl2.dylib' (no such file), '/opt/homebrew/opt/freetype/lib/libsdl2.dylib' (no such file), '/opt/homebrew/Cellar/ffmpeg/5.1.2/lib/libsdl2.dylib' (no such file), '/opt/homebrew/Cellar/libx11/1.8.1/lib/libsdl2.dylib' (no such file), '/opt/homebrew/Cellar/libxext/1.3.4/lib/libsdl2.dylib' (no such file), '/usr/local/lib/libsdl2.dylib' (no such file), '/usr/lib/libsdl2.dylib' (no such file)```

**Comment by grusell** [comment]
Both problems above should be fixed in the latest release 0.2.2 . Closing this issue.
